packages/babel-types/test/definitions.js

```diff
@@ -0,0 +1,8 @@
+var t = require("../lib");
+
+Object.keys(t.VISITOR_KEYS).forEach(function (type) {
+  try {
+    require("./definitions/" + type.toLowerCase());
+  }
+  catch (e) {}
```

Comment:
@amasad The intent was to swallow `MODULE_NOT_FOUND` errors, but this would also swallow stuff like syntax errors. Thanks for drawing attention to this. I pushed a better implementation that shouldn't swallow errors inappropriately.